python/xorbits/_mars/services/subtask/worker/processor.py

```diff
@@ -341,13 +388,40 @@ async def _store_data(self, chunk_graph: ChunkGraph):
                 storage_level,
             )
             if puts:
+                try:
+                    runner_storage: RunnerStorageActor = await mo.actor_ref(
+                        uid=RunnerStorageActor.gen_uid(self._band[1], self._slot_id),
```

Comment:
Just store the data to current RunnerStorageActor. It's OK to get the actor ref and call put_data API. The in-process actor communication is optimized to a function call automatically.

Reply:
Here we use `mo.actor_ref` to get the runner storage of the current subtask runner because RunnerStorage is currently not a property of `SubtaskProcessor` class or `SubtaskRunnerActor` class. Thus, the processors of a runner may not have direct access to the runner storage of the runner. I wonder if there's a better or more direct implementation.

Reply:
Additionally, sometimes I'm confused whether the `address` of `runner storage` should be set to the supervisor_address of the processor or the address stored in band (`band[0]`, specifically). Current implementation takes the former but I'm not quite sure about it. SubtaskRunnerActor uses `self.address` which is a property not seen in its construction function.